sdk/storage/Azure.Storage.Blobs/tests/BlobUriBuilderTests.cs

```diff
@@ -34,7 +35,7 @@ public void BlobUriBuilder_RoundTrip()
         public void BlobUriBuilder_RegularUrl_AccountTest()
```

Comment:
Can we also add a unit test using a custom domain name?

https://docs.microsoft.com/en-us/azure/storage/blobs/storage-custom-domain-name